src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -741,6 +761,10 @@ public static void setParam(final String[] args, final String confFileName) {
     INSTANCE.vmTrace =
         config.hasPath("vm.vmTrace") ? config
             .getBoolean("vm.vmTrace") : false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1807&issues=AWd4PWFsVQEMgdy30qM0&open=AWd4PWFsVQEMgdy30qM0)

[ISSUE_KEY]: AWd4PWFsVQEMgdy30qM0